### PR TITLE
Feature/te 449 show testcase in testrun tree

### PR DESCRIPTION
**Pull request by kpanier**
Showing Test case in test run view.

### CODE REVIEW

**Comment by georgportwich** [comment]
The progress information dialogs blocks the complete TE
The test execution console is showing a 404

**Comment by kpanier** [comment]
Did you executed the command:
gradlew preBuildWeb
?

File: rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/model/TestLogGroupBuilder.xtend [hunk 1]
```diff
@@ -45,10 +46,21 @@ class TestLogGroupBuilder {
 
 	def void updateCurrentEntryAfterLogGroup(String logLine, List<TestLogGroup> result) {
 		if (logLine.contains("[TE-Test:")) {
+			if (logLine.contains("Running test") &&
+				lastLine.contains(" ****************************************************")) {
```
Comment (gunther-bachmann) [inline rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/model/TestLogGroupBuilder.xtend]: We should put '[TE-Test:', 'Running test' and ' ***' in constants, too, right?
We want these constants to reside within a mutually used "component" in the long run ...

File: rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/model/TestLogGroupBuilder.xtend [hunk 2]
```diff
@@ -90,11 +102,28 @@ class TestLogGroupBuilder {
...
 				currentLogGroup = tsg
 			}
 		}
 	}
 
+	def TestLogGroupComposite getTestCaseRoot(TestLogGroup logGroup) {
+		if (logGroup.type === TestElementType.TestCase) {
+			return logGroup as TestLogGroupComposite
+		} else {
+			if (logGroup.parent === null) {
```
Comment (gunther-bachmann) [inline rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/model/TestLogGroupBuilder.xtend]: return logGroup.parent?.testCaseRoot

File: rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/model/TestLogGroupBuilder.xtend [hunk 3]
```diff
@@ -20,14 +20,15 @@ class TestLogGroupBuilder {
...
 	TestLogGroup currentLogGroup
-	
+	String lastLine
+
 	public def List<TestLogGroup> build(List<String> logLines) {
 		val result = newArrayList
 		logLines.forEach [
 			updateCurrentLogEntry(result)
 			currentLogGroup.addLogLine(it)
+			lastLine = it
```
Comment (gunther-bachmann) [inline rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/model/TestLogGroupBuilder.xtend]: put lastLine  = outside the forEach
e.g. lastLine = logLines.last

**Comment by kbayraktar** [comment]
This is the view of the TE before test execution (see Screenshot BEFORE  
before

And this is the view after test execution (see Screenshot AFTER) 
after

File: rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/model/TestLogGroupBuilder.xtend (hunk 3 above)
Comment (kpanier) [inline rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/model/TestLogGroupBuilder.xtend]: We store this information in the loop to check it in the iteration.

**Comment by kpanier** [comment]
Fixed 403 Error

**Pull request merged**